carmin-server/README.md

```diff
@@ -1,7 +1,7 @@
-# CARMIN-CBRAIN Server Implementation
+# CARMIN Server
 
 [![Build Status](https://travis-ci.org/louis-ver/CARMIN.svg?branch=develop)](https://travis-ci.org/louis-ver/CARMIN)
-[![Coverage Status](https://coveralls.io/repos/github/louis-ver/CARMIN/badge.svg?branch=setup-py)](https://coveralls.io/github/louis-ver/CARMIN?branch=setup-py)
+[![Coverage Status](https://coveralls.io/repos/github/louis-ver/CARMIN/badge.svg?branch=develop)](https://coveralls.io/github/louis-ver/CARMIN?branch=develop)
```

Comment:
I would have though about having the current branch name written here relatively, but as described here (https://github.com/github/markup/issues/913) it does not appear to be possible at the moment. We could use a pre-commit hook as a work around that would set this, but I don't think this feature is that needed.

Reply:
Let's stick with the `?branch=develop` badge for the moment, given that Coveralls always comments on the PR with the change in coverage anyways, and `develop` is what we're most concerned about. This gives us enough insight into the coverage of the project for now.